app/views/form/page.html.erb

```diff
@@ -11,11 +11,14 @@
       <% remove_other_page_errors(@log, @page) %>
       <%= f.govuk_error_summary %>
 
-      <% if @page.header.present? %>
+      <% if @page.header.present? || @page.header_partial.present? %>
         <h1 class="govuk-heading-l">
           <% if !@page.hide_subsection_label %>
             <span class="govuk-caption-l"><%= @subsection.label %></span>
           <% end %>
+          <% if @page.header_partial.present? %>
+            <% @page.header = render partial: "form/headers/#{@page.header_partial}", locals: { hholdcount: @log.hholdcount } %>
+          <% end %>
           <%= @page.header %>
```

Comment:
```suggestion
          <% if @page.header_partial.present? %>
            <%= render partial: "form/headers/#{@page.header_partial}", locals: { log: @log} %>
          <% else %>
           <%= @page.header %>
          <% end %>
```

I think I'd make this generic so that it can other types of partials can use anything in the log. In my example above I made the header rendering conditional so it's a little clearer what's happening.

Reply:
@natdeanlewissoftwire Just checking that you saw this comment!

Reply:
will come back to this if we finish this sprint's work early/when sales logs are back in scope